dts/bindings/sensor/ti,ina230.yaml

```diff
@@ -80,3 +80,8 @@ properties:
     default: 0
     # default alert limit is 0V
     description: Alert register, default matches the power-on reset value
+
+  high-precision:
```

Comment:
Looks like this belongs in `ina236.yaml` since that is the device that supports it.

Reply:
Good catch, I will move this into ina236.yaml.